src/client/effect.cpp

```diff
@@ -73,6 +74,8 @@ void Effect::draw(const Point& dest, bool drawThings, LightView* lightView)
         m_drawConductor.order = DrawOrder::FOURTH;
     }
 
+    if (drawThings &&  g_client.getEffectAlpha() < 1.f)
```

Comment:
In this case, all time will set client alpha, not better something like this?
if (drawThings &&  (g_client.getEffectAlpha() != g_client.getEffectAlpha()))